atd-vzv/src/views/nav/SideMapControlDateRange.js

```diff
@@ -189,27 +208,44 @@ const SideMapControlDateRange = ({ type }) => {
     border-radius: 4px;
     padding-left: 2px;
 
-    /* Center and enlarge picker reset button */
-    .reset-button {
-      position: relative;
-      top: 5px;
-      right: 2px;
-      width: 22px;
-      height: 22px;
-      cursor: pointer;
+    /* Center start and end date inputs */
+    [id^="start_date_"],
+    [id^="end_date_"] {
+      text-align: center;
     }
   `;
 
+  // Center and size calendar icon or button
+  const calendarInputIconStyles = `position: relative;
+    top: 2px;
+    width: 16px;
+    height: 16px;`;
+
+  const StyledCalendarIcon = styled(FontAwesomeIcon)`
+    ${calendarInputIconStyles}
+    left: 1px;
+  `;
+
+  const StyledRedoButton = styled(FontAwesomeIcon)`
+    ${calendarInputIconStyles}
+    right: 1px;
+    cursor: pointer;
+  `;
+
   return (
-    <StyledButtonContainer className="pr-0 picker-outline w-100">
+    <StyledButtonContainer className="pr-0 picker-outline">
       <DateRangePicker
         startDateId={`start_date_${type}`} // PropTypes.string.isRequired,
         endDateId={`end_date_${type}`} // PropTypes.string.isRequired,
         startDate={start} // momentPropTypes.momentObj or null,
         endDate={end} // momentPropTypes.momentObj or null,
         onDatesChange={handleDateChange} // PropTypes.func.isRequired,
         focusedInput={focused} // PropTypes.oneOf([START_DATE, END_DATE]) or null,
-        onFocusChange={(focusedInput) => setFocused(focusedInput)} // PropTypes.func.isRequired,
+        onFocusChange={(focusedInput) => {
+          setFocused(focusedInput);
+          isMobile && document.activeElement.blur(); // Do not prompt the keyboard on mobile
+        }} // PropTypes.func.isRequired,
```

Comment:
Good call ✅  there is nothing more annoying than having a keyboard you don't need taking half of the screen.